test/services/school/index.test.js

```diff
@@ -283,72 +266,78 @@ describe('school service', () => {
 			expect(result.features).to.include('rocketChat');
 		});
 
-		it('team creation by students should be updated according to environment setting'
-			+ ' without admin setting', async () => {
-			const school = await testObjects.createTestSchool({});
-
-			Configuration.set('STUDENT_TEAM_CREATION', 'enabled');
-			let result = await app.service('/schools').get(school._id);
-			expect(result.isTeamCreationByStudentsEnabled).to.be.true;
+		it(
+			'team creation by students should be updated according to environment setting' + ' without admin setting',
```

Comment:
Codacy found an issue: [Unexpected string concatenation of literals.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6074601)